src/main/java/baseball/common/Constants.java

```diff
@@ -0,0 +1,11 @@
+package baseball.common;
+
+public class Constants {
+    public static final int minNumDigit = 1;
```

Comment:
```suggestion
    public static final int MIN_NUM_DIGIT = 1;
```

매직넘버를 한 곳에 모아두셨군요!
상수가 사용 되는 곳에서 의미와 함께 선언하는 방식은 어떠신가요?!

Reply:
오 매직 넘버 관련해서 계속 고민하고 있었는데 말씀 주신 것처럼 사용 되는 곳에서 관리하게 된다면 일관되고 가독성이 좋을 것 같네요! 상수에 대한 관리 기준을 세워봐야겠습니다!! 👍